src/System.CommandLine/System/CommandLine/Enumerable.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace System.CommandLine
+{
+    // Low level replacement of LINQ Enumerable class. In particular, this implementation
+    // doesn't use generic virtual methods.
```

Comment:
Could you add a note to the comment explaining why we don't want GVMs in this library?
Thanks.